CHANGELOG.md

```diff
@@ -408,14 +408,14 @@ changelog](./docs/contributing/versioning.md#updating-changelog).
 
   ([PR #1367](https://github.com/alphagov/govuk-frontend/pull/1367))
 
-- Simplify `.govuk-main-wrapper` logic to avoid the need for large modifier in most cases.
+- Add new main wrapper modifier to avoid the need for large modifier in most cases.
```

Comment:
This change note will be removed so I've just done the bare minimum to make this history make sense.